### PR TITLE
fix logical error in download_stress function

### PR DESCRIPTION
**Pull request by xutian**
print nothing if install stress application from local

Signed-off-by: Xu Tian <xutian@redhat.com>

### CODE REVIEW

**Comment by xutian** [comment]
required by https://github.com/autotest/tp-qemu/pull/1844

File: virttest/utils_test/__init__.py [hunk 1]
```diff
@@ -2195,14 +2195,14 @@ def download_stress(self):
...
         """
-        url = self.download_url
-        tmp_path = data_dir.get_tmp_dir()
-        logging.info('Download stress tool from %s', url)
         # If it is git/wget based download proceed, else fall back if user already
         # have downloaded tool path, else raise
         try:
             download_method = getattr(
                 self, "_%s_download" % self.download_type)
+            url = self.download_url
```
Comment (balamuruhans) [inline virttest/utils_test/__init__.py]: we can use `self.download_type` instead of `url`.

**Comment by xutian** [comment]
@balamuruhans  any new comments? if no I will merge this small fix.

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m7.u7.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.7.x86_64.io-github-autotest-qemu.live_snapshot.with_payload.data_integrity: PASS 
Acked-by: Aihua Liang <aliang@redhat.com>

**Pull request merged**